packages/paste-website/src/pages/primitives/modal-dialog-primitive/index.mdx

```diff
@@ -0,0 +1,306 @@
+import {Callout, CalloutText} from '../../../components/callout';
+import {DoDont, Do, Dont} from '../../../components/DoDont';
+
+export const pageQuery = graphql`
+  {
+    allPastePrimitive(filter: {name: {eq: "@twilio-paste/modal-dialog-primitive"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/modal-dialog-primitive/"}}) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Modal Dialog Primitive"
+  categoryRoute={SidebarCategoryRoutes.PRIMITIVES}
+  githubUrl="https://github.com/twilio-labs/paste/tree/master/packages/paste-core/primitives/modal-dialog"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/primitives-modaldialog--custom-overlay-and-content"
+  data={props.data.allPastePrimitive.edges}
+/>
+
+---
+
+<contentwrapper>
+<TableOfContents headings={props.data.mdx.headings} />
+<content>
+
+## Guidelines
+
+### About the Modal Dialog Primitive
+
+i herd u like 
+The modal dialog primitive is an unstyled and barebones version of a [Modal dialog](https://www.w3.org/TR/wai-aria-practices/#dialog_modal).
+It handles the implementation details around accessibility and provides a robust API to
+build upon. For example, our Modal component (Link TBD) will be built on top of this primitive.
+If you find that our designed Modal component can’t work for your particular use case,
+we suggest falling back to this component to roll your own solution. We encourage
+using this code as a basis for all modal dialogs in order to avoid any code fragmentation
+and accessibility issues in our products.
+
+## Usage Guide
+
+### API
+
+This package is a wrapper around [`@reach/dialog`](https://reacttraining.com/reach-ui/dialog). If
+you’re wondering why we wrapped that package into our own, we reasoned that it would be best
+for our consumers’ developer experience. With reasons such as:
+
+- If we want to migrate the underlying nuts and bolts in the future, Twilio products that
+  depend on this primitive would need to replace all occurrences of `import … from ‘@reach/dialog’`
+  to `import … from ‘@some-new/package’`. By wrapping it in `@twilio-paste/modal-dialog-primitive`,
+  this refactor can be avoided. The only change would be a version bump in the package.json file.
+- We can more strictly enforce semver and backwards compatibility than some of our dependencies.
+- We can control when to provide an update and which versions we whitelist, to help reduce
+  potential bugs our consumers may face.
+- We can control which APIs we expose and how to expose them. For example, in this package we rename
+  and export only some of the source package's exports.
+
+#### Installation
+
+This package is available individually or as part of `@twilio-paste/core`.
+
+```
+yarn add @twilio-paste/modal-dialog-primitive
+```
+
+#### Usage
+
+```
```

Comment:
If this was a working example, this would be amazing too

Reply:
I added a link to the storybook just above the code snippet.  The LivePreview component doesn't seem to work well with Portals

Reply:
Too bad LivePreview and Portals don't get along for this one. We should figure out a path forward for that when we build the non-primitive modal docs.

Reply:
Does it need to be in a live preview for now? Can it just be a code block, and the example live just on the page above it? 

Or is the site not capable of dealing with it yet?